### DISCUSSION TITLE
Please share your public dashboards

### DISCUSSION THREAD

**Discussion by tailorvj** [guysoft/FullPageOS#518] [Show and tell]
Dear FullPageOS users, we'd like to showcase (and use for tests) your public dashboards. If you are running any public IoT, monitoring or whatever monitor on FullPageOS, please share it with us in the comments below and we'll use it for our tests. 

Kind regards,
Tailor

![20160609-freeboard](https://cloud.githubusercontent.com/assets/1202692/15924063/fd3f4be2-2e38-11e6-8cf4-9b036dd734fd.JPG)

**Comment by mcfrojd** [comment]
You can find many dashboards on https://www.thedash.com/explore
Even make your own (1 free)

**Comment by tailorvj** [comment]
Thanks @mcfrojd . I've just created a dashboard for FullPageOS Github repo :)
https://www.thedash.com/dashboard/BJ8GUOaN/fullscreen

**Comment by tailorvj** [comment]
Chartbeat realtime analytics dashboard demo
https://chartbeat.com/demo/

**Comment by peteruithoven** [comment]
https://github.com/peteruithoven/KickstarterStatus/

**Comment by FloppyKing** [comment]
http://empyr.net23.net/TabbedPages/

simple tabbed pages.  Built for computers at work used to display page to clock in/out and the schedule.  Also this page has a built in screensaver that kicks in after 1 minute since these are hooked to CRT monitors and didn't want them to be burned in.  Would be simple to add or alter tabs for more pages.

I know FullPageOS has fullpagedashboard built in that has similar functionality, but it had ability for user to alter tabs which we didn't want.  Also it started with cycling timer for pages which would have to manually be paused after boot since we didn't see way to disable timer and we want user to only switch to schedule page when needed since time clock page would be primary page.

**Comment by SecularSteve** [comment]
We have a Discussion Board now. In case you see this.

**Comment by guysoft** [comment]
Migrated  to a discussion
